src/scripts/main.js

```diff
@@ -1,3 +1,214 @@
 'use strict';
 
-// write code here
+const thead = document.querySelector('thead');
+const tbody = document.querySelector('tbody');
+
+// table sorting by clicking on the title (in two directions)
+const nodeList = [...tbody.children].map((i) => i.innerText.split('\t'));
+let sortByName = 'asc';
+let sortByPosition = 'asc';
+let sortByOffice = 'asc';
+let sortByAge = 'asc';
+let sortBySalary = 'asc';
```

Comment:
you can store only the last title/th index that was clicked. Then compare current clicked title with last clicked - if it's the same order is 'desc'